template/package.json

```diff
@@ -4,7 +4,7 @@
   "description": "",
   "publisher": "unknown",
   "version": "0.0.0",
-  "license": "MIT",
+  "license": "UNLICENSED",
```

Comment:
wdyt about just leaving this out, and also adding `private: true` to our generated package.json? 

we are not expecting people to publish these packages to npm right?

Reply:
It's not a valid `package.json` file without this field. Whether we default to `private: true` or not we can figure out separately from making our generate package.json adhere to the schema. I am leaning toward no, following the trail VSCode has blazed here.